File: Python/B_11720.py
"""
N개의 숫자가 공백 없이 쓰여있다. 이 숫자를 모두 합해서 출력하는 프로그램
"""

n = int(input()) # 숫자의 개수 입력받기
a = list(input()) # 숫자 입력받기
sum = 0 # 합을 저장할 변수

for i in a: # a만큼 반복
  sum += int(i) # 합 저장하기
print(sum) # 합 출력  

File: Java/src/B_11720.java
/*
 * N개의 숫자가 공백 없이 쓰여있다. 이 숫자를 모두 합해서 출력하는 프로그램 
 */
import java.util.Scanner;

public class B_11720 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		String a = sc.next();
		int sum = 0 ;
		
		for(int i = 0; i < n; i++) {
			sum += a.charAt(i) - '0'; // charAt는 char 형태 반환
		}
		System.out.println(sum);
		sc.close();
	}
}